### PR TITLE
docs: add  5.2.3 changelog

### PR DESCRIPTION
**Pull request by chenshuai2144**
## 5.2.3

`2023-02-27`

- 🐞 Fixed an issue where the Image preview icon was misaligned.[#40911](https://github.com/ant-design/ant-design/pull/40911)
- 🐞 Fixed an issue with the ConfigProvider component's form validation message execution order. [#40533](https://github.com/ant-design/ant-design/pull/40533) [@Wxh16144](https://github.com/Wxh16144)
- 🐞 Fixed an issue where the Confirm Modal `onOk` event could be triggered twice. [#40719](https://github.com/ant-design/ant-design/pull/40719) [@Rafael-Martins](https://github.com/Rafael-Martins)
- 🛠 Rewrote the `useLocale` method and exposed `localeCode` to the public. [#40884](https://github.com/ant-design/ant-design/pull/40884) [@li-jia-nan](https://github.com/li-jia-nan)
- 🐞 Fixed an issue where Segmented component items were unresponsive to mouse events. [#40894](https://github.com/ant-design/ant-design/pull/40894) [@MadCcc](https://github.com/MadCcc)
- 🛠 Refactored: replaced the LocaleReceiver component with `useLocale` and removed the LocaleReceiver component. [#40870](https://github.com/ant-design/ant-design/pull/40870) [@li-jia-nan](https://github.com/li-jia-nan)
- 🐞 Fixed an issue where the `getPopupContainer` property injected by ConfigProvider did not work. [#40871](https://github.com/ant-design/ant-design/pull/40871) [@RedJue](https://github.com/RedJue)
- 🐞 Fixed an issue where Descriptions did not accept `data-_` and `aria-_` attributes. [#40859](https://github.com/ant-design/ant-design/pull/40859) [@goveo](https://github.com/goveo)
- 🛠 Changed the Separator's DOM element from `span` to `li`. [#40867](https://github.com/ant-design/ant-design/pull/40867) [@heiyu4585](https://github.com/heiyu4585)
- 💄 Changed the component's focus `outline` to the default `4px`.[#40839](https://github.com/ant-design/ant-design/pull/40839) [@MadCcc](https://github.com/MadCcc)
- 🐞 Fixed an issue where the Badge color was displayed abnormally. [#40848](https://github.com/ant-design/ant-design/pull/40848) [@kiner-tang](https://github.com/kiner-tang)
- 🐞 Fixed an issue with the Timeline item's `className`. [#40835](https://github.com/ant-design/ant-design/pull/40835) [@Yuiai01](https://github.com/Yuiai01)
- 💄 Fixed the interaction style of the Rate component in the disabled state.[#40836](https://github.com/ant-design/ant-design/pull/40836) [@Yuiai01](https://github.com/Yuiai01)
- 🇮🇷 Added Iranian localization. [#40895](https://github.com/ant-design/ant-design/pull/40895) [@majidsadr](https://github.com/majidsadr)

---------

## 5.2.3

`2023-02-27`

- 🐞 修复 Image 预览图标不对齐的问题。[#40911](https://github.com/ant-design/ant-design/pull/40911)
- 🐞 修复 ConfigProvider 组件表单校验消息生效顺序。[#40533](https://github.com/ant-design/ant-design/pull/40533) [@Wxh16144](https://github.com/Wxh16144)
- 🐞 修复 Confirm Modal `onOk` 可能触发两次的问题。[#40719](https://github.com/ant-design/ant-design/pull/40719) [@Rafael-Martins](https://github.com/Rafael-Martins)
- 🛠 重写 `useLocale` 方法，对外暴露 `localeCode`。[#40884](https://github.com/ant-design/ant-design/pull/40884) [@li-jia-nan](https://github.com/li-jia-nan)
- 🐞 修复 Segemented 组件子项不响应鼠标事件的问题。[#40894](https://github.com/ant-design/ant-design/pull/40894) [@MadCcc](https://github.com/MadCcc)
- 🛠 重构：使用 `useLocale` 替换 LocaleReceiver 组件，并删除 LocaleReceiver 组件。[#40870](https://github.com/ant-design/ant-design/pull/40870) [@li-jia-nan](https://github.com/li-jia-nan)
- 🐞 修复 ConfigProvider 注入的 `getPopupContainer` 属性 不生效的问题。[#40871](https://github.com/ant-design/ant-design/pull/40871) [@RedJue](https://github.com/RedJue)
- 🐞 修复 Descriptions 不接受 `data-*` 和 `aria-*` 等属性的问题。[#40859](https://github.com/ant-design/ant-design/pull/40859) [@goveo](https://github.com/goveo)
- 🛠 修改 Separator 的 dom 由 `span` 改为 `li`。[#40867](https://github.com/ant-design/ant-design/pull/40867) [@heiyu4585](https://github.com/heiyu4585)
- 💄 修改组件聚焦下的 `outline` 为默认 `4px`。[#40839](https://github.com/ant-design/ant-design/pull/40839) [@MadCcc](https://github.com/MadCcc)
- 🐞 修复 Badge 颜色显示异常问题。[#40848](https://github.com/ant-design/ant-design/pull/40848) [@kiner-tang](https://github.com/kiner-tang)
- 🐞 修复 Timeline 的子项的 `className` 错误。[#40835](https://github.com/ant-design/ant-design/pull/40835) [@Yuiai01](https://github.com/Yuiai01)
- 💄 修复 Rate 在禁用状态下的交互样式。[#40836](https://github.com/ant-design/ant-design/pull/40836) [@Yuiai01](https://github.com/Yuiai01)
- 🇮🇷 增加了伊朗本地化。[#40895](https://github.com/ant-design/ant-design/pull/40895) [@majidsadr](https://github.com/majidsadr)

### CODE REVIEW

**Comment by zombieJ** [comment]
英文里需要这么多 `an issue` 么。 。 。

**Comment by chenshuai2144** [comment]
chatgpt 翻译的，我感觉还挺地道的

**Comment by kiner-tang** [comment]
> chatgpt 翻译的，我感觉还挺地道的

哈哈，原本我修复的问题并没有提issue的，竟然也加了一个`an issue`

**Comment by chenshuai2144** [comment]
我让chatgpt 帮我格式化一下，他帮我改成了统一的格式，emoji 都给换了

**Comment by zombieJ** [comment]
我意思是不要 `an issue`，太啰嗦了。

File: CHANGELOG.en-US.md [hunk 1]
```diff
@@ -15,6 +15,25 @@ timeline: true
...
+- 🛠 Refactored: replaced the LocaleReceiver component with `useLocale` and removed the LocaleReceiver component. [#40870](https://github.com/ant-design/ant-design/pull/40870) [@li-jia-nan](https://github.com/li-jia-nan)
+- 🐞 Fixed an issue where the `getPopupContainer` property injected by ConfigProvider did not work. [#40871](https://github.com/ant-design/ant-design/pull/40871) [@RedJue](https://github.com/RedJue)
+- 🐞 Fixed an issue where Descriptions did not accept `data-_` and `aria-_` attributes. [#40859](https://github.com/ant-design/ant-design/pull/40859) [@goveo](https://github.com/goveo)
+- 🛠 Changed the Separator's DOM element from `span` to `li`. [#40867](https://github.com/ant-design/ant-design/pull/40867) [@heiyu4585](https://github.com/heiyu4585)
+- 💄 Changed the component's focus `outline` to the default `4px`.[#40839](https://github.com/ant-design/ant-design/pull/40839) [@MadCcc](https://github.com/MadCcc)
+- 🐞 Fixed an issue where the Badge color was displayed abnormally. [#40848](https://github.com/ant-design/ant-design/pull/40848) [@kiner-tang](https://github.com/kiner-tang)
```
Comment (kiner-tang) [inline CHANGELOG.en-US.md:32]: 我这个其实没有issue:

```suggestion
- 🐞 Fixed the Badge color was displayed abnormally. [#40848](https://github.com/ant-design/ant-design/pull/40848) [@kiner-tang](https://github.com/kiner-tang)
```

**Comment by li-jia-nan** [comment]
#40853 #40864 这两条可以加一下

File: CHANGELOG.en-US.md [hunk 2]
```diff
@@ -15,6 +15,25 @@ timeline: true
 
 ---
 
+## 5.2.3
+
+`2023-02-27`
+
+- 🐞 Fixed where the Image preview icon was misaligned.[#40911](https://github.com/ant-design/ant-design/pull/40911)
```
Comment (zombieJ) [inline CHANGELOG.en-US.md:22]: Fix Image preview icon misaligned issue.

File: CHANGELOG.en-US.md [hunk 3]
```diff
@@ -15,6 +15,25 @@ timeline: true
...
+- 🐞 Fixed with the ConfigProvider component's form validation message execution order. [#40533](https://github.com/ant-design/ant-design/pull/40533) [@Wxh16144](https://github.com/Wxh16144)
```
Comment (zombieJ) [inline CHANGELOG.en-US.md:23]: Fix ConfigProvider validation message template override Form configure template sometime.

File: CHANGELOG.en-US.md [hunk 4]
```diff
@@ -15,6 +15,25 @@ timeline: true
...
+- 🐞 Fixed where the Confirm Modal `onOk` event could be triggered twice. [#40719](https://github.com/ant-design/ant-design/pull/40719) [@Rafael-Martins](https://github.com/Rafael-Martins)
```
Comment (zombieJ) [inline CHANGELOG.en-US.md:24]: Fixed Confirm Modal `onOk` event could be triggered twice when close.

Comment (zombieJ) [reply CHANGELOG.en-US.md:24]: 都差不多按照这类的来，where 也是不要的。

**Comment by kiner-tang** [comment]
- https://github.com/ant-design/ant-design/pull/40922

**Comment by chenshuai2144** [comment]
好啦

**Review by li-jia-nan — APPROVED** [review]

**Review by zombieJ — APPROVED** [review]

**Pull request merged**